static/scss/partials/dashboard.scss

```
@@ -138,7 +142,7 @@
 @media screen and (min-width: 768px) {
     .mpp-dashbaord-header-container {
         display: flex;
-        align-items: flex-end;
+        align-items:center;
```

With centering, does it look the same when only on one line?  Same question for three/four lines? I want to understand how the different length levels impact this layout. 

nit: whitespace

```suggestion
        align-items: center;
```